crates/aptos-rosetta/src/types/identifiers.rs

```diff
@@ -207,7 +207,7 @@ pub struct TransactionIdentifier {
 impl From<&TransactionInfo> for TransactionIdentifier {
     fn from(txn: &TransactionInfo) -> Self {
         TransactionIdentifier {
-            hash: txn.accumulator_root_hash.to_string(),
+            hash: strip_hex_prefix(&txn.hash.to_string()).to_string(),
```

Comment:
Not necessary in this PR but I wonder if the hash types should have `with_prefix` and `without_prefix` functions to make this a little easier.